tests/data_generation.py

```diff
@@ -471,6 +471,29 @@ def _exp_after_log_test_data(
             smoke_data = []
         return self.generate_tests(smoke_data, random_data)
 
+    def _to_tangent_at_identity_belongs_to_lie_algebra_test_data(
+        self, group_args_list, shape_list, n_vecs_list, belongs_atol=gs.atol
+    ):
+        """Generate data to check that to tangent at identity belongs to lie algebra.
+
+        Parameters
+        ----------
+        group_args_list : _type_
```

Comment:
the argument `shape_list` is missing from the docstring. Please make sure that the docstrings are correct before sending the PR to review 🙏

Reply:
Omg! really sorry. this will not repeat from next time for sure!